cmd/check.go

```diff
@@ -13,13 +13,13 @@ import (
 	"github.com/spf13/viper"
 )
 
-var checkCmd = &cobra.Command{
-	Use:   "check",
-	Short: "Run checks for an operator or container",
-	Long:  "This command will allow you to execute the Red Hat Certification tests for an operator or a container.",
-}
+func check() *cobra.Command {
```

Comment:
Can we use the naming `<action>Cmd` for all functions that return cobra.Command? I won't go through and mark them all.

Reply:
Can avoid using `new<action>Cmd` as I commonly see if you want, or use that instead if you prefer

Reply:
Used the 'Cmd' suffix. Good suggestion.